packages/doctor-sdk/tsconfig.json

```diff
@@ -1,6 +1,11 @@
 {
   "extends": "@rsbuild/tsconfig/base",
   "compilerOptions": {
+    "experimentalDecorators": true,
+    "moduleResolution": "node",
+    "removeComments": true,
```

Comment:
Generally, we do not remove comments or minify code when build libraries.

Reply:
may this package can rm the comments ?

Reply:
ok, i will rm this config.